BSP/Inc/BSP_UART.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2018-2023 UT Longhorn Racing Solar */
+// BSP UART header file
+
+#ifndef BSP_UART_H
+#define BSP_UART_H
+
+#include <stdint.h>
+#include "stm32f4xx_hal.h" 
+#include "stm32f4xx_hal_rcc.h"
+#include "stm32f4xx_hal_gpio.h"
+#include "stm32f4xx_hal_uart.h"
+#include "stm32f4xx_hal_usart.h"
+#include "BSP.h"
+
+#define TX_SIZE 60
+#define RX_SIZE 60
+
+typedef enum {UART_1, UART_2, UART_3, NUM_UART} UART_t;
+
+//TODO This will be replaced with general BSP status enum when it is merged in
```

Comment:
Make sure you do this eventually. Good job on adding a TODO item for it.

Reply:
Was told the general enum isn't a thing anymore, so its been replaced with our own status return types.